frontend/src/locales/scripts/en.json5

```diff
@@ -437,7 +437,36 @@
       creator: "search by creator",
       imageTitle: "Image: {title}",
       audioTitle: "Audio: {title}",
-      results: "Search results for {query}.",
+      results: {
```

Comment:
IMO, `query` should be surrounded by quotes. I don't know if the screen readers will announce quoted text differently but if they do, quotes can help identify that the phrase in quotes is what the user entered and not a part of the sentence.

Reply:
Done in https://github.com/WordPress/openverse/pull/3941/commits/db46a043bcfe4281b2f9329f30436c0b82c74cd3

I tested and can confirm SRs do announce the quotation marks.